notebooks/benchmark.py

```diff
@@ -0,0 +1,115 @@
+import os
+import simplejson as json
+from google.cloud import storage
+from google.oauth2 import service_account
```

Comment:
Could you freeze all of the required dependencies of these scripts in a `requirements.txt` file?